.github/workflows/main.yml

```diff
@@ -0,0 +1,33 @@
+on:
+  push:
+    tags:
+      - 'v*' # Push to a tag starting with a v
+
+- name: Deploy Release
+  stage: deploy
+  os: linux
+  script:
+    - make VERSION=${TRAVIS_TAG}  package
```

Comment:
Why does this work when there's no Makefile? Is an ephemeral one created for us by the GitHub Action?